### ISSUE TITLE
Feedback: "Navigate backwards to a pressed toggle button" (Toggle Button, Test 12, V23.12.14)

### ISSUE THREAD

**Issue by jha11y** [w3c/aria-at#1029]
## Description of Behavior

VoiceOver says "selected" instead of "pressed".  I did not fail this, but technically the assertion does not pass as written. Would automated tests pass this correctly?

## Test Setup

- Test File: [test-12-navBackToPressedToggleButton-voiceover_macos.collected.html](https://aria-at.netlify.app/tests/toggle-button/test-12-navBackToPressedToggleButton-voiceover_macos.collected.html)
- AT: VoiceOver for macOS (Version 14.2)
- Browser: Safari (Version 17.2.1)

**Comment by mcking65** [comment]
Closing based on discussion on Jan 25, 2024.
